### ISSUE TITLE
Set up static files with S3

### ISSUE THREAD

**Issue by cdriehuys** [knowmetools/tutorial-site#23]
Static files and media files should be stored on S3.

**Comment by cdriehuys** [comment]
We are blocked on this while waiting on feedback from jrief/django-sass-processor#62. If we wanted to work around this, we could introduce another task runner such as gulp to build our SASS files.

**Comment by cdriehuys** [comment]
We now use gulp to manage assets, so this is no longer a blocking issue.
